### PR TITLE
Fix: Regtest config

### PR DESCRIPTION
**Pull request by bguiz**
## What

- Cleaned up the config file used by regtest
- Added a util file to quickly determine if the RSKj regtest node has websockets exposed properly

## Why

- The forked version of ganache was unable to connect to the RSKj regtest running in docker

### CODE REVIEW

File: docker/regtest.conf [hunk 1]
```diff
@@ -3,24 +3,25 @@ blockchain.config.name = "regtest"
...
+                hosts: ["localhost"],
+                port: 4444,
+            },
+            ws: {
+                enabled: true,
+                bind_address: "0.0.0.0",
+                hosts: ["localhost"],
+                port: 4445,
+            },
+        },
```
Comment (chrisli30) [inline docker/regtest.conf]: Strict JSON wouldn't allow trailing comma, better remove those ,

File: docker/websocket-test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+var WebSocketClient = require('websocket').client;
+
```
Comment (chrisli30) [inline docker/websocket-test.js:2]: Nice

**Review by chrisli30 — APPROVED** [review]

File: docker/regtest.conf (hunk 1 above)
Comment (bguiz) [reply docker/regtest.conf]: done

File: docker/websocket-test.js (hunk 2 above)
Comment (bguiz) [reply docker/websocket-test.js:2]: credit goes to ignacio!

**Review by chrisli30 — APPROVED** [review]

**Pull request merged**